### PR TITLE
EZP-30326: As a user I want to see if my search in UDW has no results

### PR DESCRIPTION
**Pull request by tischsoic**
JIRA: https://jira.ez.no/browse/EZP-30326

**Requires: https://github.com/ezsystems/ezplatform-admin-ui/pull/920**

## Screenshot
Screen Shot 2019-03-22 at 11 08 04

### CODE REVIEW

File: src/modules/universal-discovery/components/search/css/search.component.css [hunk 1]
```diff
@@ -45,7 +45,8 @@
 }
 
 .c-search__submit[disabled] {
-    opacity: 0.75;
+    opacity: 0.3;
+    cursor: not-allowed;
```
Comment (tischsoic) [inline src/modules/universal-discovery/components/search/css/search.component.css:49]: Note: I slightly changed styles of:
Screen Shot 2019-03-22 at 11 08 31
to match:
Screen Shot 2019-03-22 at 11 08 41
both in disabled state.

File: src/modules/universal-discovery/components/bookmarks/bookmarks.component.js [hunk 2]
```diff
@@ -41,7 +36,6 @@ export default class BookmarksComponent extends Component {
                 perPage={bookmarksPerPage}
                 contentTypesMap={contentTypesMap}
                 title={tableTitle}
-                noItemsMessage={noBookmarksMessage}
```
Comment (dew326) [inline src/modules/universal-discovery/components/bookmarks/bookmarks.component.js:44]: Why bookmark tables will not have this message anymore?

File: src/modules/universal-discovery/components/search/search.component.js [hunk 3]
```diff
@@ -130,21 +133,77 @@ export default class SearchComponent extends Component {
...
-    render() {
+    renderSearchTips() {
+        const { items } = this.state;
+
+        if (items === null || items.length) {
+            return null;
+        }
+
+        const searchTipsTitle = Translator.trans(/*@Desc("Some helpful search tips:")*/ 'search.tips.headline', {}, 'search');
+        const searchTip1 = Translator.trans(/*@Desc("Check spelling of keywords.")*/ 'search.tips.check_spelling', {}, 'search');
```
Comment (dew326) [inline src/modules/universal-discovery/components/search/search.component.js]: Please, never ever use a variable named `text{number}`

File: src/modules/universal-discovery/components/search/search.component.js [hunk 4]
```diff
@@ -130,21 +133,77 @@ export default class SearchComponent extends Component {
...
+                onItemSelect={onItemSelect}
+                perPage={searchResultsPerPage}
+                contentTypesMap={contentTypesMap}
+                title={tableTitle}
+                selectedContent={selectedContent}
+                onSelectContent={onSelectContent}
+                canSelectContent={canSelectContent}
+                onItemRemove={onItemRemove}
+                multiple={multiple}
+                noItemsMessage={`Sorry, no results were found for "${lastSearchText}".`}
```
Comment (dew326) [inline src/modules/universal-discovery/components/search/search.component.js]: Translation

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/modules/universal-discovery/components/bookmarks/bookmarks.component.js (hunk 2 above)
Comment (tischsoic) [reply src/modules/universal-discovery/components/bookmarks/bookmarks.component.js:44]: It was never used in Bookmarks, it must have stayed after some modifications in the past.

**Review by dew326 — APPROVED** [review]

**Review by katarzynazawada — APPROVED** [review]

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by tischsoic** [comment]
Merged up:
- master: https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/627bb8d51293d43f517ed546f37384bcd023a57d

**Pull request merged**